_posts/2020-08-11-backports-application.adoc

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: From zero to hero on backporting pull requests
+date: 2020-08-11
+tags: user-story
+author: gastaldi
+---
+= From zero to hero on backporting pull requests
+
+The Quarkus project moves fast and when we prepare bugfix releases, we usually have several dozens of pull requests to backport.
+The number of pull requests to backport is enormous and (as it was aforementioned) doing it via the GitHub UI is inconvenient and takes a lot of time (click PR, copy/paste commit hashes to cherry-pick, remove label, assign milestone, assign milestone to all fixed issues, next) plus some limitation of the UI (it's not possible to sort by merge date to avoid conflicts, for example).
+So we decided to automate this work, and built an application.
+Of course, with Quarkus!
+
+Before we dig into the solution, let me give you a quick explanation about our release process.
```

Comment:
"explanation about" -> "explanation of"?

Reply:
And generally someone should probably review this from grammar perspective. I see some things that I believe are tiny mistakes, but I'm not a native speaker, so... :-)